lib/rubocop/cop/rails/compact_blank.rb

```diff
@@ -33,6 +35,10 @@ module Rails
       #   collection.delete_if { |_k, v| v.blank? } # Same behavior as `Array#compact_blank!` and `Hash#compact_blank!`
       #   collection.reject!(&:blank?)              # Same behavior as `ActionController::Parameters#compact_blank!`
       #   collection.reject! { |_k, v| v.blank? }   # Same behavior as `ActionController::Parameters#compact_blank!`
+      #   collection.select!(&:present?)            # Same behavior as `Array#compact_blank!` and `Hash#compact_blank!`
+      #   collection.select! { |_k, v| v.present? } # Same behavior as `Array#compact_blank!` and `Hash#compact_blank!`
```

Comment:
These don't appear to be compatible with `Array#compact_blank!` and `Hash#compact_blank!`.

```ruby
[].select!(&:present?)            #=> nil
[].select! { |_k, v| v.present? } #=> nil
[].compact_blank!                 # => []

{}.select!(&:present?)            #=> nil
{}.select! { |_k, v| v.present? } #=> nil
{}.compact_blank!                 # => {}
```